examples/animation-controls/animation-controls.js

```diff
@@ -0,0 +1,62 @@
+
+const animationNames = {
+  attack: 'Armature|TRex_Attack',
+  death: 'Armature|TRex_Death',
+  idle: 'Armature|TRex_Idle',
+  jump: 'Armature|TRex_Jump',
+  run: 'Armature|TRex_Run',
+  walk: 'Armature|TRex_Walk',
+};
+
+updateAnimationMixer = () => {
+
+  const data = {}
+  data.clip = "none"
+  Object.entries(animationNames).forEach((name) => {
+
+    const el = document.getElementById(name[0])
+    
+    if (el.checked) {
+      data.clip = name[1]
+    }
+  })
+
+  const keys = ["duration",
+                "clampWhenFinished",
+                "crossFadeDuration",
+                "loop",
+                "repetitions",
+                "timeScale",
+                "startAt"]
+  keys.forEach((key) => {
+    const el = document.getElementById(key)
+    var value = el.value
```

Comment:
For new code where you use `const`/`let`, please don't use `var`. Replace the `var` by `let` here.
For strings, can you please use single quotes or double quotes everywhere, and not a mix of the two? It seems we mainly use single quotes in this repo.

Reply:
@diarmidmackenzie there was just this one last comment above and you can merge.

Reply:
Thanks for the reminder on this.  Will make the fix and merge soon.